source/projects/overview.txt

```diff
@@ -0,0 +1,50 @@
+.. _rm-project-overview:
+
+=======================
+User Interface Overview
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+When you open a project in Relational Migrator, the project user interface displays.
+You can manage many aspects of your Relational Migrator project from this interface:
```

Comment:
(minor wording)
Consider something similar to the below. Reasoning is that this set people up to understand what they're supposed to do with the information on the page. 

"Review the UI element descriptions to understand how you can use the UI to manage the project look and feel, actions, and capabilities."

Reply:
Slight tweak to the suggested:

```
Review the UI element descriptions to understand how you can use the UI 
to manage the project appearance, functionality, and capabilities
```